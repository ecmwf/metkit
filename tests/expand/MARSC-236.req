retrieve,
accuracy=12,area=90.0/0.0/-90.0/359.5,date=20240102,domain=g,grid=0.5/0.5,leve=off,levtype=sfc,padding=0,param=134/137/165/166/167/168/235,stream=da,style=dissemination,time=00,type=an,
target="data.reference"