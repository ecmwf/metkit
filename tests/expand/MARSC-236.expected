RETRIEVE,
    CLASS      = OD,
    TYPE       = AN,
    STREAM     = OPER,
    EXPVER     = 0001,
    REPRES     = GG,
    LEVTYPE    = SFC,
    PARAM      = 134/137/165/166/167/168/235,
    DATE       = 20240102,
    TIME       = 0000,
    STEP       = 0,
    DOMAIN     = G,
    TARGET     = data.reference,
    RESOL      = AV,
    ACCURACY   = 12,
    STYLE      = DISSEMINATION,
    AREA       = 90/0/-90/359.5,
    GRID       = .5/.5,
    PADDING    = 0 